hyperspace/core/src/macros.rs

```diff
@@ -1036,25 +1036,25 @@ macro_rules! chains {
 				}
 			}
 
-			pub fn wasm_code_hash(&self) -> Option<CodeHash> {
-				let maybe_code_hash = match self {
+			pub fn wasm_checksum(&self) -> Option<CodeHash> {
```

Comment:
This should return `Option<Checksum>`

Reply:
Wow, good catch! I will fix it immediately.